x-pack/legacy/plugins/siem/public/components/embeddables/map_tool_tip/map_tool_tip.tsx

```diff
@@ -64,7 +64,7 @@ export const MapToolTipComponent = ({
             getLayerName(layerId),
           ]);
 
-          setFeatureProps(featureProperties);
+          setFeatureProps((featureProperties as unknown) as FeatureProperty[]);
```

Comment:
Commented on a potential cleanup here https://github.com/elastic/kibana/pull/61264#discussion_r398851856, but since this involves additional SIEM refactoring I can take care of this in a follow PR, so feel free to leave as is. 👍